pkg/rancher-desktop/config/settings.ts

```diff
@@ -380,7 +383,7 @@ function parseSaveError(err: any) {
  * most changes will get picked up from the defaults.
  */
 const updateTable: Record<number, (settings: any) => void> = {
-  1: (settings) => {
+  1: (settings: any) => {
```

Comment:
Hmm, the `any` here should be redundant (because the type of `updateTable` dictates what it is).

Reply:
true